src/test/test-execute.c

```diff
@@ -76,6 +76,11 @@ static void test(Manager *m, const char *unit_name, int status_expected, int cod
         check(m, unit, status_expected, code_expected);
 }
 
+static int in_container(void) {
+        char *container = getenv("container");
+        return container != NULL && strcmp(container, "systemd-nspawn") == 0;
```

Comment:
We have a helper function for that: `streq_ptr`. So you could write:
```
   return streq_ptr(getenv("container"), "systemd-nspawn");
```
Nevertheless the condition that is tested doesn't match the function name. I don't think the test needs to be so specific, why not just use `!strempty(getenv("container"))`.